packages/runtime/container-runtime/src/garbageCollectionTombstoneUtils.ts

```diff
@@ -26,11 +26,6 @@ export function sendGCTombstoneEvent(
     if(throwOnTombstoneUsage) {
         mc.logger.sendErrorEvent(event, error);
     } else {
-        mc.logger.sendTelemetryEvent(
-            {
-                ...event,
-                ...error,
-            }
-        );
+        mc.logger.sendTelemetryEvent(event, error);
```

Comment:
Great, thanks!  This also matters because some properties on the error object may not be ok to log, and the logging code takes care of skipping those.